client/src/pages/Meals.js

```diff
@@ -0,0 +1,36 @@
+// TEMPORARY PAGE WITH RECIPES WILL BE REMOVED WHEN CHEF PROFILE PAGE IS CREATED
+import React from "react";
+import ResponsiveSideBar from "../components/ResponsiveSideBar";
+import Main from "../components/Main";
+import { Typography } from "@material-ui/core";
+import MealCard from "../components/MealCard";
+import meals from "../lib/mockedMeals";
+
+function Meals(props) {
+    return (
+        <React.Fragment>
+            <ResponsiveSideBar>
+                <Typography>Responsive Side Bar</Typography>
+            </ResponsiveSideBar>
+            <Main>
+                {meals.map((meal) => {
+                    return (
+                        <MealCard
+                            key={meal.id}
+                            id={meal.id}
```

Comment:
Can we just pass the whole meal rather than each field individually?

Reply:
Yes!